src/docs/product/issues/issue-details/performance-issues/main-thread-io.mdx

```diff
@@ -1,30 +1,30 @@
 ---
-title: "IO on Main Thread for Mobile"
+title: "File IO on Main Thread for Mobile"
 sidebar_order: 20
 description: "Learn more about File IO on Main Thread and how to diagnose and fix these issues."
 ---
 
 <Include name="early-adopter-note.mdx" />
 
-The Main UI thread in a mobile application handles user interface events such as button presses and page scrolls. If the Main UI thread performs long running operations like File IO, it will block the whole UI until the operation is complete, preventing the user from interacting with the app. If the Main UI thread is blocked for a long period of time, it can cause App Hangs and Application Not Responding errors.
+The Main UI thread in a mobile application handles user interface events such as button presses and page scrolls. If the Main UI thread performs long-running operations like File IO, it will block the whole UI until the operation is complete, preventing the user from interacting with the app. If the Main UI thread is blocked for a long time, it can cause App Hangs and Application Not Responding errors.
```

Comment:
```suggestion
The main UI thread in a mobile application handles user interface events such as button presses and page scrolls. To prevent App Hangs and Application Not Responding errors, the main UI thread should not be used for performing long-running operations like File IO. These kinds of operations block the whole UI until they finish running and get in the way of the user interacting with the app.
```